.github/workflows/test-gh.yml

```diff
@@ -20,6 +20,12 @@ jobs:
     runs-on: ubuntu-latest
     environment: DockerHub E2E
     steps:
+    - name: Free Disk Space (Ubuntu)
+      uses: jlumbroso/free-disk-space@main
```

Comment:
Can we change this to be a specific sha? just to make sure we are always using the same version.

Reply:
Switched to using the latest released version instead of main.